docs/docs/sourcing-from-buttercms.md

```diff
@@ -49,22 +49,25 @@ Make sure to add your **API_TOKEN** from your dashboard. In this guide you will
 
 ```javascript:title=gatsby-config.js
 module.exports = {
-{
+  plugins: [
+    {
       resolve: `gatsby-source-buttercms`,
       options: {
-        authToken: `your_auth_token`,
-        // Optional. Returns values for the supplied content field keys.
+        authToken: `<API_TOKEN>`,
+        // Optional array of Collection key
```

Comment:
It looks like this comment now explains only the value of `keys` vs the entire `contentFields` object. If we're making this more generic, is there any reason we shouldn't use the sample code in the ButterCMS Gatsby quickstart?

```
module.exports = {
{
      resolve: `gatsby-source-buttercms`,
      options: {
        authToken: `your_api_token`,
        // Optional. Returns values for the supplied content field keys.
        contentFields: {
          keys: [`collection_key`],
          // Optional. Set to 1 to enable test mode for viewing draft content.
          test: 0,
        },
        // Optional. Array of page types.
        pageTypes: [`page_type_key`],
        // Optional array of locales (if configured in your account)
        locales: [`en`, `es`, `fr`]
      },
    },
}
```

Reply:
Hi @AishaBlake !  By Quickstart, are you referring to https://github.com/ButterCMS/gatsby-starter-buttercms ?